velox/type/ShortDecimal.h

```diff
@@ -57,9 +60,21 @@ struct ShortDecimal {
     return unscaledValue_ >= other.unscaledValue_;
   }
 
+  ShortDecimal& operator+=(const ShortDecimal& other) {
+    unscaledValue_ += other.unscaledValue_;
+    return *this;
+  }
+
  private:
   int64_t unscaledValue_;
 };
+
+static inline ShortDecimal operator*(
```

Comment:
Why is this operator defined differently from the + operator?

Reply:
Apparently, only unary operators can be defined inside the class. Binary operators must be defined outside.

Reply:
Thank you for clarifying. Would you add a comment explaining why we need to define this operator? It is not obvious because the operator is defined only for 32-bit integers.

Reply:
This requirement is coming from
https://github.com/facebookincubator/velox/blob/cb32115022a6c6e2ced31ebb23b9196da5b28dab/velox/functions/prestosql/aggregates/SumAggregate.h#L87